tools/base/fpgainfo/board.c

```diff
@@ -413,7 +419,7 @@ fpga_result fpgainfo_board_info(fpga_token token)
 	if (print_board_info) {
 		res = print_board_info(token);
 	} else {
-		OPAE_ERR("No print_board_info entry point:%s\n", dlerror());
+		OPAE_MSG("No print_board_info entry point:%s\n", dlerror());
```

Comment:
I thought OPAE_ERR was appropriate for these three, since you're returning an error code.

Reply:
Some platforms like MCP doesn't support board info, fpgainfo out put shows error message.